spectre/ddtbd.ml

```diff
@@ -484,13 +484,15 @@ module Detector(Machine : Primus.Machine.S) = struct
     Machine.Global.put models m >>= fun () ->
     Machine.Local.put state s >>= fun () ->
     Machine.List.iter step3 ~f:(fun h3 ->
-        let taints = Set.union h3.cnd_taint h3.off_taint |>
-                     Set.to_sequence in
-        let taints = if get all_models then taints
-          else Seq.take taints 1 in
-        Machine.Seq.iter taints ~f:(fun t ->
-            Object.to_value t >>= fun v ->
-            Machine.Observation.make was_detected (v,h3)))
+      Set.to_sequence h3.cnd_taint |>
```

Comment:
Can you please use ocp-indent to reindent the code?

Reply:
Done